File: easy/java/FuncCall.java
import common.VulnChild;
import iface.VulnRunner;
import cmd_exec.InterfaceImpl;

public class FuncCall {
    public static void runByChild(String cmd) throws Exception {
        VulnChild child = new VulnChild();
        child.exec(cmd);
    }

    public static void runByInterface(String cmd) throws Exception {
        VulnRunner runner = new InterfaceImpl();
        runner.run(cmd);
    }
}

File: easy/kotlin/FuncCall.kt
import common.VulnChild
import cmd_exec.InterfaceImpl
import iface.VulnRunner

object FuncCall {
    fun runByChild(cmd: String) {
        val child = VulnChild()
        child.exec(cmd)
    }

    fun runByInterface(cmd: String) {
        val runner: VulnRunner = InterfaceImpl()
        runner.run(cmd)
    }
}
